CHANGES.md

```diff
@@ -0,0 +1,86 @@
+# Revision history for sDDF
+
+## Release 0.5.0
+
+### General
+
+* Move to Microkit 1.4.0.
+    * Previously a development version of Microkit was used, now we use
+      an official release for everything.
+* Add QEMU support to the timer and echo server examples.
+* Remove dependency on a full libc, all the provided components use our own
+  utility functions now.
+* Moved all components and examples to use a 'Makefile snippets' system
+  to make it easier to put together sDDF components in a system.
+* Introduced snippets for the Zig build system, as an alternative to Make.
+    * You can find instructions for using the Zig build system in each example's
+      README.
+    * No components are solely built with Zig, the primary build system is
+      still Make.
+
+### Block
+
+* Move the capacity of the shared queue out of shared memory
+  and into the queue handles. This prevents malicious clients
+  from messing with the queue size and potentially causing the
+  virtualiser or other trusted components to crash/misbehave.
+* Introduce a config header to enable the system designer to have a variable
+  amount of clients, previously it was all hard-coded.
+    * It should be noted that we are working on tooling to automate this process
+      and these configs are likely to significantly change by the next release.
+* Change block virtualiser to use offsets instead of raw addresses and do the
```

Comment:
> Change block virtualiser to use offsets instead of raw addresses and do the offset to DMA physical addresses in the virtualiser. 

Seems like a contradiction

Reply:
Yes I need to reword this better.